src/global-shortcuts.c

```diff
@@ -394,6 +395,7 @@ shortcuts_listed_cb (GObject *source_object,
           results = g_variant_ref_sink (g_variant_builder_end (&results_builder));
         }
 
+      response = 0;
```

Comment:
These tends to be in the impl method call, e.g. https://github.com/flatpak/xdg-desktop-portal/blob/8255fabc37cd6a943c07a0b469e5a1a944aeaecf/data/org.freedesktop.impl.portal.ScreenCast.xml#L250 and https://github.com/flatpak/xdg-desktop-portal/blob/8255fabc37cd6a943c07a0b469e5a1a944aeaecf/data/org.freedesktop.impl.portal.Inhibit.xml#L85.

There hasn't been a release yet, so we can still add the `response` argument where it's missing.

Reply:
Addressed. Added a couple of extra changes of things that cropped up when testing it.